### ISSUE TITLE
Sources.list is not refreshed on update

### ISSUE THREAD

**Issue by aviks** [JuliaLang/WinRPM.jl#58]
The `sources.list` file was updated in this commit: https://github.com/JuliaLang/WinRPM.jl/commit/cb12a4136e9afb2a158ff888bc8ea81d3e72e619

When I updated WinRPM today via a `Pkg.update()`, after having tried to do a `Pkg.add("ZMQ")` earlier which had loaded WinRPM and set its global variable `sources

**Comment by tkelman** [comment]
Did you have any local modifications making the package dirty? I don't think there's much to do here on this, though backporting https://github.com/JuliaLang/julia/pull/13506 should help.

**Comment by aviks** [comment]
No, there were no local modifications at all. This was a relatively new julia install. Yes, JuliaLang/julia#13506 will fix this.

**Comment by aviks** [comment]
One way to fix it could be to read the `sources.list` on every WinRPM.update() call. Currently, it is only read on a `__init_()` and stored as a `global const`
